modules/serverless-applications-checking-revision-status.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/debugging-serverless-applications.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-applications-checking-revision-status_{context}"]
+= Checking Revision status
+
+You might need to check the status of your revision to identify the issue with your Serverless application.
+
+.Procedure
+
+. If you configure your route with a `Configuration` object, run the following command to get the name of the `Revision` object created for you deployment:
++
+[source,terminal]
+----
+$ oc get configuration <configuration-name> --output jsonpath="{.status.latestCreatedRevisionName}"
+----
++
+You can find the configuration name in the `Route.yaml` file.
```

Comment:
@mgencur Is this sufficient explanation? I've also linked to the relevant section in OCP docs. https://github.com/openshift/openshift-docs/pull/72687/commits/136210ce5ea5c776d1ceff4240de14dc343eec12

Reply:
It's better. Could we mention that this is an "OpenShift Route"? 
I was confused because there's also a Route in Knative itself (route.serving.knative.dev), see https://github.com/knative/specs/blob/main/specs/serving/knative-api-specification-1.0.md#route